pkg/networkserver/mac/utils.go

```diff
@@ -571,6 +571,18 @@ outer:
 	return chs
 }
 
+const defaultClassBCDownlinkInterval = time.Second
+
+func DeviceClassBCDownlinkInterval(dev *ttnpb.EndDevice, defaults ttnpb.MACSettings) time.Duration {
+	if t := dev.GetMacSettings().GetClassBCDownlinkInterval(); t != nil {
```

Comment:
For devices that don't have this field in the registry, does this just return nil and we skip to `L583`?

Or do existing devices need a migration to set this field in the DB and then set a default value? 

This is just for me since I don't know much about how non existent fields are handled when you get them from Redis. cc: @adriansmares